v4l2-request-test.c

```diff
@@ -279,6 +307,10 @@ int main(int argc, char *argv[])
 			config.drm_driver = strdup(optarg);
 			break;
 		case 's':
+			free(config.slices_path);
```

Comment:
We didn't have any config.slices_path allocated at this point, did we? Unless I'm mistaken, this should be removed.

Reply:
If you are not going to go with the default (which will pick ./data/<preset-name>), and you are selecting -s the following argument will let to the path where the slice-data are stored. At least, this is the idea.

Reply:
Yes but this is done later in the code, so config.slices_path was not allocated yet. I would expect this to crash actually.